assets/js/blocks/rating-filter/block.tsx

```diff
@@ -66,16 +66,19 @@ const translations = {
 /**
  * Component displaying a rating filter.
  *
- * @param {Object}  props            Incoming props for the component.
- * @param {Object}  props.attributes Incoming block attributes.
- * @param {boolean} props.isEditor   Whether the component is being rendered in the editor.
+ * @param {Object}  props                 Incoming props for the component.
+ * @param {Object}  props.attributes      Incoming block attributes.
+ * @param {boolean} props.isEditor        Whether the component is being rendered in the editor.
+ * @param {boolean} props.noRatingsNotice Element displayed in the editor if there's no ratings to display.
```

Comment:
The types in the JSDoc seem incorrect. `noRatingsNotice` is defined below as `ReactElement | null` while here is `boolean`. I think you can safely remove the JSDoc type information (and just leave the description) as the types will be correctly inferred by TypeScript.